python/foglamp/plugins/north/pi_server/pi_server.py

```diff
@@ -799,28 +796,45 @@ def _generate_omf_typename_automatic(self, asset_code):
                         status_code = resp.status
                         text = await resp.text()
 
-            except Exception as e:
-                _error = Exception(plugin_common.MESSAGES_LIST["e000024"].format(e))
-                _message = plugin_common.MESSAGES_LIST["e000024"].format(e)
+            except (TimeoutError, asyncio.TimeoutError) as ex:
+
+                # The conversion to a string of the exception TimeoutError produces an empty string,
+                # so it forces a proper message
+                if str(ex) == "":
+                    details = type(ex).__name__
+                else:
+                    details = str(ex)
```

Comment:
you can simply remove now these 4 lines and 2 lines of comment, say connection timed out in logs!

Reply:
done